pkg/agent/agent_linux.go

```diff
@@ -208,7 +211,7 @@ func (i *Initializer) BridgeUplinkToOVSBridge() error {
 		return err
 	}
 	// Create uplink port.
-	uplinkPortUUId, err := i.ovsBridgeClient.CreateUplinkPort(uplink, config.UplinkOFPort, map[string]interface{}{"antrea-uplink": "true"})
+	uplinkPortUUId, err := i.ovsBridgeClient.CreateUplinkPort(uplink, config.UplinkOFPort, map[string]interface{}{"antrea-uplink": "true", interfacestore.AntreaInterfaceTypeKey: interfacestore.AntreaUplink})
```

Comment:
Should we deprecate "antrea-uplink"?

Reply:
@gran-vmv is it possible to use the Antrea interface type instead of the "antrea-uplink"? May I know where is "antrea-uplink" used in the existing code?

Reply:
It's used in start_ovs for ensuring the uplink is removed before stopping ovs.

Reply:
Yes, it is used to check the existence of uplink when OVS shutdown, you can refactor here if you want to remove existing key-value.
https://github.com/antrea-io/antrea/blob/eded51a28f8e58359ca61e560bf756504b01c800/build/images/scripts/start_ovs#L114